### ISSUE TITLE
audio cue Settings not included in settingsLayout.ts:

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#205391]
1. Open the settings UI
2. Open dev tools console
3. 🐛 see:

```
WARN SettingsEditor2: Settings not included in settingsLayout.ts: audioCues.chatRequestSent, audioCues.chatResponsePending, audioCues.chatResponseReceived, audioCues.clear, audioCues.debouncePositionChanges, audioCues.diffLineDeleted, audioCues.diffLineInserted, audioCues.diffLineModified, audioCues.enabled, audioCues.format, audioCues.lineHasBreakpoint, audioCues.lineHasError, audioCues.lineHasFoldedArea, audioCues.lineHasInlineSuggestion, audioCues.lineHasWarning, audioCues.noInlayHints, audioCues.notebookCellCompleted, audioCues.notebookCellFailed, audioCues.onDebugBreak, audioCues.save, audioCues.taskCompleted, audioCues.taskFailed, audioCues.terminalBell, audioCues.terminalCommandFailed, audioCues.terminalQuickFix, audioCues.volume
```
